src/wallet/rpcwallet.cpp

```diff
@@ -2406,6 +2406,11 @@ static UniValue getwalletinfo(const JSONRPCRequest& request)
             "  \"paytxfee\": x.xxxx,                (numeric) the transaction fee configuration, set in " + CURRENCY_UNIT + "/kB\n"
             "  \"hdseedid\": \"<hash160>\"            (string, optional) the Hash160 of the HD seed (only present when HD is enabled)\n"
             "  \"private_keys_enabled\": true|false (boolean) false if privatekeys are disabled for this wallet (enforced watch-only wallet)\n"
+            "  \"scanning\":                        (json objects) current scanning details, or null if no scan is in progress\n"
+            "    {\n"
+            "      \"duration\" : xxxx              (numeric) elapsed seconds since scan start\n"
+            "      \"progress\" : x.xxxx,           (numeric) scanning progress percentage [0.0, 1.0]\n"
+            "    }\n"
             "}\n"
```

Comment:
Nice! Am indeed seeing (in testnet):
```
"scanning": {
    "duration": 168,
    "progress": 0.265890921329112
  }
```
However, after running `abortrescan`, this is output:
```
"scanning": null
```
Is this the desired behavior? Note that before rescanning, or after aborting the rescan, stopping and restarting bitcoind, this field is absent.

Reply:
(Might be good to add a test that defines the desired behavior).

Reply:
> Is this the desired behavior?

Yes, this was suggested by @MarcoFalke. `scanning` key is also present right?

BTW, `"progress": 0.265890921329112` maybe should limit precision?

Reply:
> (Might be good to add a test that defines the desired behavior, and describe it in the commit message).

I can test the key is there, but I don't think theres's a reliable way to test while rescanning.

Reply:
> `scanning` key is also present right?

No scanning key returned in those cases during my manual testnet testing.

> BTW, `"progress": 0.265890921329112` maybe should limit precision?
 
Perhaps to hundredths of a percent? Trying to imagine how a wallet might use it.

Reply:
Precision is fine. We use the same for `  "verificationprogress": 0.9831896495042335,`

Reply:
> No scanning key returned in those cases during my manual testnet testing.

Are you sure you are running the right binary? All code paths should push back a "scanning" key?

Reply:
> Are you sure you are running the right binary? All code paths should push back a "scanning" key?

Argh, you are right about the binary. Now seeing `"scanning": null` in the cases above. Apologies.

Reply:
Adding the expected behavior to the commit message and minimal test would still be good. I'll volunteer to do the test if anyone wants it.